server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/FlowInstanceController.java

```diff
@@ -246,4 +260,42 @@ public SuccessResponse<PartitionPlanConfig> getPartitionPlan(@PathVariable Long
         return Responses.ok(this.partitionPlanScheduleService.getPartitionPlanByFlowInstanceId(id));
     }
 
+    private List<FlowInstanceDetailResp> dispatchCreateSchedule(CreateFlowInstanceReq createReq) {
```

Comment:
please don't add any logic at controller layer

Reply:
move to the scheduleService